test/unit/rounding/fixed_point/elastic_integer/elastic_fixed_point.cpp

```diff
@@ -112,4 +112,81 @@ namespace {
                 -10),
                 "");
     }
+
+    namespace efxp_round_test{
```

Comment:
Many of these namespaces -- whether they have obscure `efxp` or verbose `elastic_fixed_point` -- are tautological: it's already clear from the filename in which they are defined that they relate to `elastic_fixed_point`. No need to add such repetition.

Reply:
Ok. I will review these. Namespace names do conflict. If one is using very simple namespace name such as "multiply_test" you may have naming clashes even though they are in different files. I did get this kind of compilation error with my non imaginative T0, T1, T2 and T3 datatypes. Thus I put them into specific namespaces. Thus the unnecessary repetition.